### ISSUE TITLE
hello,Have you used the HIC test dataset

### ISSUE THREAD

**Issue by Ron-zs** [NhatHoang2002/ToXCL#4]
There is no IHC test dataset in the project folder, and it was not found in the code. Did IHC not have a test dataset and SBIC use one?

**Comment by NhatHoang2002** [comment]
Hi, the IHC dataset does not have a test set so we created a validation set ourself (see Appendix A).

**Comment by Ron-zs** [comment]
Okay, received. Has your paper been published in a conference or journal?

**Comment by NhatHoang2002** [comment]
Yes, the paper is already published at NAACL 2024.

**Comment by Damazti** [comment]
> Hi, the IHC dataset does not have a test set so we created a validation set ourself (see Appendix A).

Hello, may I ask whether the IHC validation set in the directory is the one ultimately used for performance evaluation, or is it intended for hyperparameter tuning?
